network/network_windows.go

```diff
@@ -185,6 +190,55 @@ func (nm *networkManager) newNetworkImplHnsV1(nwInfo *NetworkInfo, extIf *extern
 	return nw, nil
 }
 
+// add ipv4 and ipv6 routes (if dual stack mode) to windows Node
+func (nm *networkManager) addNewNetRules(nwInfo *NetworkInfo) error {
+	var (
+		err error
+		out string
+	)
+
+	// get interface name of VM adapter
+	ifName := nwInfo.MasterIfName
+	if !strings.Contains(nwInfo.MasterIfName, ifNamePrefix) {
+		ifName = fmt.Sprintf("%s (%s)", ifNamePrefix, nwInfo.MasterIfName)
+	}
+
+	// add ipv4 and ipv6 new net rules to windows node
+	for _, subnet := range nwInfo.Subnets {
+		prefix := subnet.Prefix.String()
+		gateway := subnet.Gateway.String()
+
+		ip, _, _ := net.ParseCIDR(prefix)
```

Comment:
Any reason why we are not acknowledging the error from parseCIDR

Reply:
check is added